src/main/java/mpern/sap/commerce/build/util/Stopwatch.java

```diff
@@ -0,0 +1,21 @@
+package mpern.sap.commerce.build.util;
+
+public class Stopwatch {
+
+    long start;
+
+    long duration = -1;
+
+    public Stopwatch() {
+        start = System.currentTimeMillis();
```

Comment:
shouldn't this be `nanoTime? but let's not bikeshed too much

Reply:
Changed to nanotime (although we do not need the precision). stop and duration methods still return milliseconds as it is more than enough.